src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -4,13 +4,17 @@
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
 import java.nio.file.Path;
+import java.util.List;
 import java.util.function.Predicate;
 import java.util.logging.Logger;
 
 import javafx.collections.ObservableList;
 import javafx.collections.transformation.FilteredList;
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
```

Comment:
Then this way you also don't need to have the imports from the logic package into the model package.